install/config/nginx.dist

```diff
@@ -52,6 +52,10 @@ server {
 		return 403;
 	}
 
+	location /cache {
+		expires 1y;
```

Comment:
If simplecache is off, these requests will be served by PHP. I don't know if ngnix will overwrite the headers sent by our CacheHandler.